### PR TITLE
Loop experiments

### PR DESCRIPTION
**Pull request by sikorka**
Hello Miron 😄 Could you review? Thank you 👏

### CODE REVIEW

File: gameproject/src/main/java/com/qatrain/janushgame/model/Board.java [hunk 1]
```diff
@@ -18,36 +18,36 @@
...
-     * Setting algorithm which can create random Grid using static values inside method
+     * Setting algorithm which can create random Board using static values inside method
      */
-    public void createGrid() {
-        System.out.println("Creating grid...");
+    public void createBoard() {
+        System.out.println("Creating board...");
 
-        rows = (int) (Math.random() * ROWS_MAX) + ROWS_MIN; //Random function from Math class returns the value between 0 and 1, so we have to use algorithm to create Grid with 3 <= size <= 16
+        rows = (int) (Math.random() * ROWS_MAX) + ROWS_MIN; //Random function from Math class returns the value between 0 and 1, so we have to use algorithm to create Board with 3 <= size <= 16
```
Comment (miron-kuzniar) [inline gameproject/src/main/java/com/qatrain/janushgame/model/Board.java]: I just realized that this algorithm may give us values > than `ROWS_MAX` ie. (1 * 7) + 3 = 10. 
We should change this like this:
`rows = (int) (Math.random() * (ROWS_MAX - ROWS_MIN) + ROWS_MIN`

File: gameproject/src/main/java/com/qatrain/janushgame/model/Board.java [hunk 2]
```diff
@@ -18,36 +18,36 @@
...
         columns = (int) (Math.random() * COLUMNS_MAX) + COLUMNS_MIN; //The same as above
```
Comment (miron-kuzniar) [inline gameproject/src/main/java/com/qatrain/janushgame/model/Board.java]: The same as above.

**Comment by miron-kuzniar** [comment]
@sikorka 

Everything looks good except small change in Board:) Please check my code comment.

**Comment by miron-kuzniar** [comment]
[As a user I can see Board](https://trello.com/c/VrOG3t5D/17-as-a-user-i-can-see-board)

**Comment by sikorka** [comment]
Thank you for the review 👍  

Comments are applied + additional changes: 
- renamed module `gameproject` to `janushgame` 
- added packaging to jar in `pom.xml` + `README.md` and added how to run app from command line 
- commented `--` line printing `drawBoard()` so that shape of the board resembles dimensions 

Sorry for so many files changed. Could you pls review :)

**Comment by miron-kuzniar** [comment]
@sikorka 

Great job. This game is awesome :D
Thanks!!!

**Pull request merged**